### PR TITLE
Accept Feature Functionality

### PR DESCRIPTION
**Pull request by MahmoudMuhammedAli**
- add the ability for users to accept or ignore a feature
- add the act of accepting or ignoring a feature to the history graph for the ability to undo/redo
- Tags are now representative of the feature they are pointing to 
- add correct SVG on the OSM pins to make it apparent for the users what the pin is for

### CODE REVIEW

File: modules/services/mapillary.js [hunk 1]
```diff
@@ -32,7 +33,46 @@ let _mlyShowFeatureDetections = false;
...
+let _dataset;
 
+const rapidTypes = ['object--support--utility-pole', 'object--street-light', 'object--bench' ,'object--bike-rack', 'object--fire-hydrant' ];
+const tag_dict = {
+    'object--fire-hydrant': {
+        emergency: 'fire_hydrant'
+    },
+    'object--support--utility-pole': {
+        power: 'pole',
+        'man_made': 'utility_pole'
```
Comment (cbeddow) [inline modules/services/mapillary.js]: most of the keys in here and not in quotes but 'man_made' is -- not sure which is correct but should that be fixed?

Comment (Bonkles) [reply modules/services/mapillary.js]: I'd remove the quotes from the member identifier, but leave them in for the string type values. 

e.g. `'emergency' : 'fire_hydrant'`

to `emergency: 'fire_hydrant'`

File: modules/services/mapillary.js [hunk 2]
```diff
@@ -275,24 +339,22 @@ export default {
     // Get visible map (point) features
     mapFeatures: function(projection) {
         const limit = 5;
-        const ret = searchLimited(limit, projection, _mlyCache.points.rtree);
-        return ret;
+        return searchLimited(limit, projection, _mlyCache.points.rtree);
     },
+
     // Get filtered Map (points) features (utility-pole, street-light, bench, bike-rack, fire-hydrant)
     filteredMapFeatures: function(projection) {
```
Comment (Bonkles) [inline modules/services/mapillary.js]: If I recall correctly, `filteredMapFeatures()` no longer gets called anymore. If this is still the case we should delete this method!

**Review by Bonkles — CHANGES_REQUESTED** [review]

File: modules/services/mapillary.js (hunk 2 above)
Comment (Bonkles) [reply modules/services/mapillary.js]: Looks gewd otherwise- a couple of smol requests/suggestions

**Review by Bonkles — APPROVED** [review]

**Pull request merged**